frontend/packages/kubevirt-plugin/package.json

```diff
@@ -37,7 +37,8 @@
       "kubevirt-gating": [
         "integration-tests/tests/example.scenario.ts",
```

Comment:
we can remove the example test case from here. 
the vm.action.base test is already enough as a basic test